README.md

```diff
@@ -45,6 +45,13 @@ git push
 
 Given that github is limiting the number of API requests for unauthenticated users, it'd be a good idea to use your personal access token. You can create a personal token in [Github settings](https://github.com/settings/tokens). Generate New Token -> Click on a repo -> Create an environment variable called GITHUB_TOKEN and provide your token as the value.
 
+The token requires the following permissions:
+* `repo:status`
+* `public_repo`
```

Comment:
repo:public_repo

Reply:
@nikita-b I have referenced them so that the string can be searched for in the page when creating the token

Reply:
```
'x-oauth-scopes': 'public_repo, read:user, repo:status'
```
This is what the API call shows